v0/11-1-0_Adding_outcomes.markdown

```diff
@@ -2,13 +2,15 @@
 
 > To add an {{outcome}} record, go to {{Work}}->Add->{{Outcome}}s on the main menu. 
 
-To add an {{outcome}} record, go to {{Work}}->Add->{{Outcome}}s on the main menu. You will see the main screen for entering {{outcome}} records:
+To add an {{outcome}} record, go to {{Work}}->Add New->{{Outcome}}s on the main menu. You will see the main screen for entering {{outcome}} records:
 
 ![Adding an {{outcome}} record]({{imgpath}}74a.png)
 
 The first two tabs, 'When and where' and 'attendance' are standard (see [11.1.1  Adding {{outcome}}s - when and where](/help/index/v/{{version}}/p/11.1.1) and [11.1.2  Adding {{outcome}}s - attendance](/help/index/v/{{version}}/p/11.1.2)): the remaining tabs are the names of the {{outcome}} categories you (or your system administrator) have set up. It is also possible to customise the system to add extra fields and tabs, so your system may differ to the example here. See section [18.4.0  Custom {{work}} and other record tabs and fields](/help/index/v/{{version}}/p/18.4.0) for more information about customising {{outcome}} records.
 
-When you have added all information in all the tabs, click the 'save' button in the bottom right. 
+In some systems, the attendance tab in Outcomes may be called 'Respondent'.  This operates in a simpler fashion.  Simply enter the name of the referrer, referree and who the referral to to.
```

Comment:
This depends on the global setting  - do we want to link to an explanation of this? Maybe as a second-pass edit.
Also, the 'name of the referrer...' - this is in the outcomes page so the labels are 'Outcome about user' and 'Outcome recorded with/by staff'.